build/util/src/lib.rs

```diff
@@ -215,3 +235,52 @@ fn toml_from_env<T: DeserializeOwned>(var: &str) -> Result<Option<T>> {
         .context("deserializing configuration")?;
     Ok(Some(rval))
 }
+
+pub fn build_notifications() -> Result<()> {
+    let out_dir = out_dir();
+    let dest_path = out_dir.join("notifications.rs");
+    let mut out = std::fs::File::create(dest_path)?;
+
+    let full_task_config = task_full_config_toml()?;
+
+    if full_task_config.notifications.len() >= 32 {
```

Comment:
Should this be 31? The 32nd notification will collide with `INTERNAL_TIMER_NOTIFICATION`.

Reply:
I think this is correct, because it's `>=` (so 32 notification exactly will trigger the conditional)

Reply:
Oy, yep, you're right. :man_facepalming: